pkg/sloop/processing/relationship.go

```diff
@@ -0,0 +1,126 @@
+package processing
+
+import (
+	"encoding/json"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/pkg/errors"
+	"github.com/salesforce/sloop/pkg/sloop/kubeextractor"
+	"github.com/salesforce/sloop/pkg/sloop/store/typed"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+)
+
+
+
+
+//  transfer watchRec.Payload to Relationship payload: Convert watchRec to record in relationship table and save key-value pair
+func updateRelationshipTable(tables typed.Tables, txn badgerwrap.Txn, watchRec *typed.KubeWatchResult, metadata *kubeextractor.KubeMetadata) error {
+	if watchRec.Kind != kubeextractor.PodKind {
+		glog.Infof("Skipping relationship table update as kubewatch result is not PodKind")
```

Comment:
do we know how often this is going to happen? this could be very noisy in the logs. Consider putting a `.V(10)` so the normal logs don't have this but we can get it if we need it.

Reply:
please don't use a constant `10`, use a named literal instead.

Reply:
updated